client/src/components/todo/DatePicker.vue

```diff
@@ -21,10 +21,9 @@ function onDateInput(event: Event) {
 }
 
 const formattedDate = computed(() => {
-  const date = dateValue.value?.toLocaleDateString('en-GB');
-  const year = date?.split('/')[2];
-  const month = date?.split('/')[1];
-  const day = date?.split('/')[0];
-  return `${year}-${month}-${day}`;
+  if (dateValue.value) {
```

Comment:
Instead of if you can do something like this dateValue.value?.toISOString()?.slice(0, 10) || ''